src/components/inherited-map/models/root-store.ts

```diff
@@ -36,12 +36,12 @@ const RootStore = types
     });
     const loadArea = () => {
       const { center, zoom } = self.mapStore;
-      self.areaStore.loadArea(center, zoom);
+      void self.areaStore.loadArea(center, zoom);
```

Comment:
Resolves `"@typescript-eslint/no-floating-promises"`. We want to be explicit that we ignore the returned promise as this helps reading the logic flow and catch bugs.